module/cpf.sh

```diff
@@ -26,17 +29,22 @@ declare -A COLORS=(
 )
 
 #. cpf:module -={
-function ::cpf:module_is_modified() {
+function ::cpf:module_is_unchanged() {
```

Comment:
Okay, so two things:
1. Your first comment can be resolved by changing it to use the variables `${TRUE}` and `${FALSE}`; those are synonymous with `${CODE_SUCCESS}` and `${CODE_FAILURE}`; but make sense in the context of a boolean function.
2. Your second comment is void, `is_modified` is as much of a question as `is_unchanged`.  So adding a change like this just confuses things.